bin/cqlsh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+# Copyright (C) 2023-present ScyllaDB
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+here=$(dirname "$0")
+exec "$here/../tools/cqlsh/bin/cqlsh" "$@"
```

Comment:
how about having a tiny script in `/scripts`, so we can just source it for setting up the alias / functions or `$PATH` ? developers can even source this script in their `$HOME/.bash_profile`.

Reply:
I don't mind it. Personally I prefer stateless environments (that don't rely on me to source some script), but it's okay to have one for people who want it.

Reply:
I also prefer stateless environments. I have over 10 ScyllaDB trees checked out on my work machine. Which one do I source the env script from? :)

Reply:
So do I. Please refer to virtualenv.